java/flight/flight-sql-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ITDriverJarValidation.java

```diff
@@ -113,7 +105,7 @@ public void validateShadedJar() throws IOException {
    * allowed prefixes
    *
    * @param name the jar entry name
-   * @throws AssertionException if the entry is not allowed
+   * @throws AssertionError if the entry is not allowed
```

Comment:
Just curious, how this change comes from? I assume the Jupiter API is using this?

Reply:
The `AssertionError` is a built-in class in Java. 

- [AssertionError](https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F/java/lang/AssertionError.html)

The previous code had a typo; it actually used `AssertionError`, but the comment described it as `AssertionException`. That's why I corrected it.

https://github.com/apache/arrow/blob/9f0101ec14336b2baad45d57320fb56c71d9321b/java/flight/flight-sql-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ITDriverJarValidation.java#L117-L140